package/src/components/Link/v1/Link.test.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
+import Link from "./Link";
+
+test("basic snapshot", () => {
+  const component = renderer.create(
+    <Link href="http://google.com" onClick={(event) => alert("clicked")}>
+      <img src="/reaction-design-system-logo.svg" width="200" height="200" />
+    </Link>
+  );
+
+  const tree = component.toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
I'd also add at least add a simple `onClick` test using enzyme. [Here](https://gist.github.com/nnnnat/fc035b7f96b39b2b099fc178ee529fb7) is a simple example test that should be all you need for this comp.